### ISSUE TITLE
Focus state should be consistent for all keyboard focus(able) elements

### ISSUE THREAD

**Issue by Ngoakor12** [Ngoakor12/coding-resource-finder#185]
**Describe the bug**

The focus state is not consistent

**To Reproduce**

Focus state should be consistent for all elements

**Screenshots**

example of correct focus state:
![image](https://user-images.githubusercontent.com/54069197/224612971-7bccc5b1-3d6d-481a-bb7d-d01163f95faa.png)

example of wrong focus state:
![image](https://user-images.githubusercontent.com/54069197/224613102-4240f454-4796-4c1a-af11-333d5d1a7ca1.png)

**Desktop (please complete the following information):**
 

**Smartphone (please complete the following information):**
 

**Additional context**

**Comment by rajat4984** [comment]
Hey, Ngoakor12 I think I can fix this. Can you assign it to me ? Thanks

**Comment by Ngoakor12** [comment]
@rajat4984 thanks for showing interest. Unfortunately someone has already open a PR. I'll let you know if it doesn't work out

**Comment by rajat4984** [comment]
@Ngoakor12 okay
